src/components/Navigation.js

```diff
@@ -0,0 +1,40 @@
+import { NavLink } from 'react-router-dom';
+import ScreenReaderOnly from './ScreenReaderOnly';
+import { HiHome } from 'react-icons/hi';
```

Comment:
Same here for your icon names as mentioned above. Check also your other files. :-)